### PR TITLE
Allow using `wazuh-db` as `component` in API agents endpoints to get active configuration

### PR DESCRIPTION
**Pull request by mcarmona99**
|Related issue|
|---|
| #14229  |

This PR closes https://github.com/wazuh/wazuh/issues/14229

In this pull request, I have added the `wazuh-db` string to the enum used for the `component` variable. The `component` variable is the one used in the endpoints:

```
/agents/{agent_id}/config/{component}/{configuration}
/cluster/{node_id}/configuration/{component}/{configuration}
/manager/configuration/{component}/{configuration}
```

I have also updated the framework functions in charge of getting the components' active configuration.

Unit tests and API integration tests were also updated. The API integration tests are going to fail due to https://github.com/wazuh/wazuh/issues/13929, but I have performed them in my localhost with the conflictive tests commented:

```
test_agent_GET_endpoints.tavern.yaml - Cluster environment 
	 92 passed, 93 warnings
	 
test_manager_endpoints.tavern.yaml - Cluster environment 
	 31 passed, 32 warnings

test_cluster_endpoints.tavern.yaml - Cluster environment 
	 45 passed, 46 warnings
```

### CODE REVIEW

File: api/test/integration/test_agent_GET_endpoints.tavern.yaml [hunk 1]
```diff
@@ -1898,6 +1898,42 @@ stages:
...
+    request:
+      verify: False
+      url: "{protocol:s}://{host:s}:{port:d}/agents/000/config/wazuh-db/internal"
+      method: GET
+      headers:
+        Authorization: "Bearer {test_login_token}"
+    response:
+      status_code: 200
+      json:
+        error: !anyint
```
Comment (vicferpoy) [inline api/test/integration/test_agent_GET_endpoints.tavern.yaml:1911]: Why can't we use specific error numbers here?

File: api/test/integration/test_agent_GET_endpoints.tavern.yaml [hunk 2]
```diff
@@ -1898,6 +1898,42 @@ stages:
             rootcheck:
               sleep: !anyint
 
+  - name: Request-Wazuh-db-Internal
```
Comment (vicferpoy) [inline api/test/integration/test_agent_GET_endpoints.tavern.yaml:1901]: This name format doesn't match the standard. I don't know why we're maintaining it.

**Review by vicferpoy — CHANGES_REQUESTED** [review]

File: api/test/integration/test_agent_GET_endpoints.tavern.yaml (hunk 1 above)
Comment (mcarmona99) [reply api/test/integration/test_agent_GET_endpoints.tavern.yaml:1911]: I have changed from `!anyint` to error numbers in all the test cases.

File: api/test/integration/test_agent_GET_endpoints.tavern.yaml (hunk 2 above)
Comment (mcarmona99) [reply api/test/integration/test_agent_GET_endpoints.tavern.yaml]: I have updated the format of all names

**Review by AdriiiPRodri — APPROVED** [review]

**Pull request merged**